.github/workflows/test_osdk.yml

```diff
@@ -10,9 +10,13 @@ on:
   push:
     branches:
       - main
+  # Schedule to run on every day at 15:30 UTC (23:30 CST)
```

Comment:
I am wondering why the time of scheduled tests are specified in CST (Central Standard Time). Isn't the CI server based in China? @grief8

Reply:
You are right. CI server is located in China. However, GitHub triggers the scheduling event according to Central Standard Time.

Reply:
I see. Then, we should use both CST and Beijing time as UTC is irrelevant.

Reply:
I thought CST here means China Standard Time...
The cron expressions I found use the UTC time zone by default, so we should use UTC and Beijing time?
```yaml
  # Schedule to run on every day at 15:30 UTC (23:30 Beijing Time)
  schedule:
    - cron: '30 15 * * *'
```

From Wiki:

> The official national standard [time](https://en.wikipedia.org/wiki/Time) is called Beijing Time (BJT, [simplified Chinese](https://en.wikipedia.org/wiki/Simplified_Chinese_characters): 北京时间; [traditional Chinese](https://en.wikipedia.org/wiki/Traditional_Chinese_characters): 北京時間; [pinyin](https://en.wikipedia.org/wiki/Pinyin): Běijīng shíjiān) domestically because it is based on the [120th meridian east](https://en.wikipedia.org/wiki/120th_meridian_east), where [Beijing City](https://en.wikipedia.org/wiki/Beijing) is located,[[1]](https://en.wikipedia.org/wiki/Time_in_China#cite_note-1)[[2]](https://en.wikipedia.org/wiki/Time_in_China#cite_note-2) and China Standard Time (CST) internationally.

Reply:
> I thought CST here means China Standard Time...

Yeah... I thought the C stands for China, too. I don't know I googled "CST" and the first result is Central Standard Time.

> However, GitHub triggers the scheduling event according to Central Standard Time.

@grief8 Are you really sure that CST stands for Central Standard Time? Any reference? There are a number of time zones in the US. Why does Github prefer CST? This is just weird.

Reply:
> I thought CST here means China Standard Time... The cron expressions I found use the UTC time zone by default, so we should use UTC and Beijing time?
> 
> ```yaml
>   # Schedule to run on every day at 15:30 UTC (23:30 Beijing Time)
>   schedule:
>     - cron: '30 15 * * *'
> ```
> 
> From Wiki:
> 
> > The official national standard [time](https://en.wikipedia.org/wiki/Time) is called Beijing Time (BJT, [simplified Chinese](https://en.wikipedia.org/wiki/Simplified_Chinese_characters): 北京时间; [traditional Chinese](https://en.wikipedia.org/wiki/Traditional_Chinese_characters): 北京時間; [pinyin](https://en.wikipedia.org/wiki/Pinyin): Běijīng shíjiān) domestically because it is based on the [120th meridian east](https://en.wikipedia.org/wiki/120th_meridian_east), where [Beijing City](https://en.wikipedia.org/wiki/Beijing) is located,[[1]](https://en.wikipedia.org/wiki/Time_in_China#cite_note-1)[[2]](https://en.wikipedia.org/wiki/Time_in_China#cite_note-2) and China Standard Time (CST) internationally.

I mean we should use whatever time zone that cron uses. But in the comment, express the time in the timezone of our Server. Because we want to choose an early-in-the-morning time for our CI server to have the best network connectivity.